ui/sections/system/inspectors/boot-pool.reel/boot-pool.js

```diff
@@ -89,17 +87,16 @@ exports.BootPool = AbstractInspector.specialize(/** @lends BootPool# */ {
 
     _populateComponentIfNeeded: {
         value: function () {
+            var self = this;
             if (!this._populatingPromise && (!this.bootEnvironments || !this.bootVolume)) {
                 return this._populatingPromise = Promise.all([
-                    this._bootEnvironmentService.list(),
-                    this._bootEnvironmentService.getBootVolumeConfig(),
+                    this._sectionService.getBootVolumeConfig(),
                     this._sectionService.listDisks()
```

Comment:
Because we just want to be sure we got the disks from the middleware before continuing.
There may be some more changes on the way we feed the UI with data, but it will occur post-RELEASE ;)